README.md

```diff
@@ -1,23 +1 @@
-# mini-ecommerce-site
-
-## INSTRUCTIONS ON HOW TO FORK THIS REPO
-1) Click on `Fork`
-2) Scroll down to the botton and click on `create a fork`
-3) Scroll back up and click on code and then copy the link to the repo
-
-### On your local workspace (Your Computer)
-1) Create a folder
-2) cd into your folder in `GitBash` CLI (You can right-click your working directory and click on GitBash Here to do this)
-3) Once in your folder type the following command
-   `git clone https://github.com/<YOUR-GITHUB-NAME>/mini-ecommerce-site.git` Replace this <YOUR-GITHUB-NAME> with your GitHub Name
-4) You can now start working on the project and practise using the following commands:
-
-   #### To add files in the staging area, making them ready for the next commit
-   Use the command:  `git add .`
-   
-   ####  To create a snapshot of what changes you have made before pushing them to the remote repository
-   Use the command: `git commit -m 'commit message'`
-   
-   #### To upload changes made in your local repository to the remote repository
-   Use the command: `git push origin master`
```

Comment:
Hi. You now need to edit this readme file and Include 
-the instructions on how to run the the code.
-technologies used to develop the site
-The name of the developer (Your name)
You have to start developing the site based on the given design.